services/site/tests/components/homepage/ChallengeHeader.spec.tsx

```diff
@@ -46,16 +41,4 @@ describe("ChallengeHeader", () => {
     expect(userAccountModalApi.show).toHaveBeenCalledTimes(1);
     expect(userAccountModalApi.show).toHaveBeenCalledWith("signup");
   });
-
-  it("renders the github button", () => {
```

Comment:
why did you remove these two tests?

Reply:
Testing if the buttons are present do not provide any additional value to checking if the sign up button is rendered if the user is logged in.
In my opinion we would have to test some kind of behaviour here to justify having additional tests.